Documentation/api.md

```diff
@@ -202,6 +205,44 @@ NamespaceSelector is a selector for selecting either all namespaces or a list of
 
 [Back to TOC](#table-of-contents)
 
+## PodMonitor
+
+PodMonitor defines monitoring for a set of pods.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| metadata | Standard object’s metadata. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata | [metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#objectmeta-v1-meta) | false |
+| spec | Specification of desired Pod selection for target discrovery by Prometheus. | [PodMonitorSpec](#podmonitorspec) | true |
+
+[Back to TOC](#table-of-contents)
+
+## PodMonitorList
+
+PodMonitorList is a list of PodMonitors.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| metadata | Standard list metadata More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata | [metav1.ListMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#listmeta-v1-meta) | false |
+| items | List of PodMonitors | []*[PodMonitor](#podmonitor) | true |
+
+[Back to TOC](#table-of-contents)
+
+## PodMonitorSpec
+
+PodMonitorSpec contains specification parameters for a PodMonitor.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| jobLabel | The label to use to retrieve the job name from. | string | false |
+| targetLabels | TargetLabels transfers labels on the Kubernetes Service onto the target. | []string | false |
```

Comment:
this should just be what `podTargetLabels` is no?

Reply:
Yep. Will change